README.md

```diff
@@ -130,6 +130,14 @@ chmod-socket = 660
 vacuum = true
 ```
 
+#### Docker
+
+*GnuCash Web* can be run using [Docker](https://www.docker.com/), either using the published 
+[packages](https://github.com/joshuabach/gnucash-web/packages), DockerFile, or using [docker compose](https://docs.docker.com/compose/) 
+with the provided sample `docker-compose.yml` files.
```

Comment:
Do you want this in the readme or in the docker-compose file? I'd argue that the rest of the documentation is sufficient to understand the docker-compose configuration, unless you want to explain how the Docker Compose plugin itself works alongside the Postgres image. For the general Dockerfile, I could add a table describing the possible environment variables if that is desired. 

There shouldn't be any difference compared to regular setup other than what's required to run and connect to Docker, which depends on the individual setup and network/firewall configuration.

Reply:
I mean in the README. This does not have to be verbose, just some pointers on what needs to be done apart from running `docker-compose up`.

For example, someone not familiar with docker might struggle on how to connect to the database directly, since it does not map a port. This should be briefly explained.

Reply:
I mean, it can be as simple as:

---
*GnuCash Web* can be run using [Docker](https://www.docker.com/), either using the published 
[packages](https://github.com/joshuabach/gnucash-web/packages), DockerFile, or using [docker compose](https://docs.docker.com/compose/) 
with the provided sample `docker-compose.yml` file. This file uses a simple SQLite backend. A file `docker-compose-postgres.yml' with a PostgreSQL backend is also provided. 

In either case, configuration is done via environment variables in the compose file instead of the default configuration file. The same options are available.

To access the postgres database, expose the postgres port `5432` on a host port (think about security before doing that) in the compose file [add link to explanation in docker docs] or connect to the container on that port directly.

Note that while the SQLite version includes a sample database, the PostgreSQL is uninitialized, see [below]() [add link] for more information.
---

Or something similar in that spirit.